packages/yoroi-extension/app/ergo-connector/components/signin/SignTxPage.js

```diff
@@ -126,20 +126,41 @@ class SignTxPage extends Component<Props> {
   }
 
   // Tokens can be minted inside the transaction so we have to look it up there first
-  _resolveTokenInfo: TokenEntry => $ReadOnly<TokenRow> = tokenEntry => {
+  _resolveTokenInfo: TokenEntry => $ReadOnly<TokenRow> | null  = tokenEntry => {
     const { tx } = this.props;
     const mintedTokens = mintedTokenInfo(tx);
     const mintedToken = mintedTokens.find(t => tokenEntry.identifier === t.Identifier);
     if (mintedToken != null) {
       return mintedToken;
     }
-    return this.props.getTokenInfo(tokenEntry);
+
+    try {
+      return this.props.getTokenInfo(tokenEntry);
+    } catch (error) {
+      return null
+    }
```

Comment:
@AhmedIbrahim336 , also, this seems like too wide of an assumption. There might be various reasons for an error and here we are silently ignoring it.

if you check what is being passed as the `getTokenInfo` prop, you will see:
```
genLookupOrFail(this.generated.stores.tokenInfoStore.tokenInfo)
```

Which just wraps a simple map to explicitly throw an error in case the key is not found. So instead of catching that error - you can replace the wrapper with something like `genLookupOrNull` and not throw the error in the first place.